### DISCUSSION TITLE
建议在任务开始的时候就提draft pr

### DISCUSSION THREAD

**Discussion by 0x01toby** [labring/sealos#1221] [Ideas] [answered]
github doc: https://github.blog/2019-02-14-introducing-draft-pull-requests/

建议在接到任务的时候，就提好draft pr：
- 可以很方便的看到目前pr/任务的状态
- 更早的开始review 代码，更早的发现问题，更早的讨论问题

**Comment by cuisongliu** [comment, marked as answer]
不错，赞成 替换了WIP的模式

**Comment by zzjin** [comment]
可以尝试在代码里面增加一下spec的目录来表述draft的需求和实现 这样更充分的讨论点?
